app/models/user.rb

```diff
@@ -7,6 +7,14 @@ class User < ActiveRecord::Base
   has_many :reviews
   has_many :airlines
 
+  def self.admins
+    User.where(admin: true)
+  end
+
+  def self.non_admins
+    User.where(admin: false)
+  end
```

Comment:
Do you prefer doing these methods as class methods rather than scopes? These look more like scopes to me.

example:
```
scope :admins ->, { where(admin: true) }
```